samcli/commands/_utils/template.py

```diff
@@ -87,6 +87,9 @@ def move_template(src_template_path, dest_template_path, template_dict):
     # update any relative paths in the template to be relative to the new location.
     modified_template = _update_relative_paths(template_dict, original_root, new_root)
 
+    # if a stack only has image functions, the directory for that directory won't be created.
```

Comment:
Why are we doing this? As in Image Functions are already working (however they are working) right?

Reply:
for image function, sam build won't generate any artifact files, instead it is a image tag (from docker).